### PR TITLE
Revert old cache rules

### PR DESCRIPTION
**Pull request by nboisteault**
No more necessary since https://github.com/3liz/lizmap-web-client/pull/2729

### CODE REVIEW

**Comment by nboisteault** [comment]
@laurentj I'm not sure about this change : https://github.com/3liz/lizmap-web-client/pull/2852/files#diff-2507844269c2b3d15c0d54a58d870595caca6fc3bcfad6e5e23e708204ee6833L26
Should it be handled by Lizmap backend or the web server?
With our docker test stack if I remove this line the asset is not cached and there is no etag. Maybe because there is `Content-Type: text/javascript;charset=UTF-8` and not `Content-Type: text/javascript` in header?

**Comment by laurentj** [comment]
@nboisteault as it is a php script, there is no cache by default, so we should set it. However, one year of cache is too high. So we should not remove it, but change its value.

File: lizmap/modules/view/controllers/translate.classic.php [hunk 1]
```diff
@@ -23,7 +23,6 @@ public function index()
         $rep = $this->getResponse('binary');
         $rep->mimeType = 'text/javascript';
         $rep->doDownload = false;
-        $rep->setExpires('+1 year');
```
Comment (laurentj) [inline lizmap/modules/view/controllers/translate.classic.php:26]: we should not remote it, but set a lower duration

**Review by laurentj — CHANGES_REQUESTED** [review]

File: lizmap/modules/view/controllers/translate.classic.php (hunk 1 above)
Comment (laurentj) [reply lizmap/modules/view/controllers/translate.classic.php:26]: 1 week is a good duration I think

**Comment by laurentj** [comment]
I merge the PR, and will fix the cache duration, in order to have this patch in the release of the week

**Pull request merged**